bundles/binding/org.openhab.binding.plclogo/src/main/java/org/openhab/binding/plclogo/internal/PLCLogoBinding.java

```diff
@@ -212,14 +208,14 @@ protected void execute() {
                         }
 
                         if (config.isSet()) {
```

Comment:
Hmmm... Since logo support shorts only, Integer.MAX_VALUE is really invalid. Assignment will be done in any case. And the "magic" is used on initialization of lastValue only. You are right: my logic is more "hidden". Btw: MAX_INT is not really magic. It's the same constant as Integer.MAX_VALUE, but for C++